packages/react-components/src/organisms/NewsCard.tsx

```diff
@@ -90,21 +84,17 @@ const NewsCard: React.FC<NewsCardProps> = ({
   ) : (
     <Headline4>{title}</Headline4>
   );
-
+  const newsLink = text && news({}).article({ articleId: id }).$;
   return (
     <Card>
       <div css={cardStyle}>
-        <div css={imageContainerStyle}>
-          {thumbnail ? (
-            <img
-              alt={`"${title}"'s thumbnail`}
-              src={thumbnail}
-              css={[imageStyle]}
-            />
-          ) : (
-            placeholders[type]
-          )}
-        </div>
+        <ImageLink
+          containerPropStyle={imageContainerStyle}
```

Comment:
I really don't like injected styles. It means if someone changes <ImageLink> it will be affected here but it isn't easy to find usages of. It should be avoided wherever possible